templates/rack_mini_profiler.rb

```diff
@@ -0,0 +1,5 @@
+if ENV["RACK_MINI_PROFILER"].to_i > 0
```

Comment:
According to [the docs](https://github.com/MiniProfiler/rack-mini-profiler#configuration-options):

> toggle_shortcut (default Alt+P) - a jquery.hotkeys.js-style keyboard shortcut, used to toggle the mini_profiler's visibility. See https://github.com/jeresig/jquery.hotkeys for more info.